<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_adm</class>
 <widget class="QMainWindow" name="MainWindow_adm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Adm_tab">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>90</y>
      <width>791</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="Book_alter">
     <attribute name="title">
      <string>图书信息管理 </string>
     </attribute>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>书名</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>类别名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="book_bname_edit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="bcat_name_edit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="book_search">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="book_add">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>30</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>新增</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="book_table">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>671</width>
        <height>231</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>20</number>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>书号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>书名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>分类名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>库存</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>位置</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出版日期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>作者</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="book_save">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>320</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Stu_alter">
     <attribute name="title">
      <string>学生信息管理</string>
     </attribute>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>学号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stu_sno_edit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="stu_sname_edit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="stu_search">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stu_add">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>新增</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="stu_table">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>671</width>
        <height>231</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>10</number>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>学号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>已借数量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>诚信等级</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>性别</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>专业</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="stu_save">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>320</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Rec">
     <attribute name="title">
      <string>记录浏览</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>671</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_rec">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>681</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>--------记录号----------书号-----------书名-----------库存------------位置------------</string>
      </property>
     </widget>
     <widget class="QPushButton" name="book_rec">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>图书管理记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stu_rec">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>学生管理记录</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Person_info">
     <attribute name="title">
      <string>个人信息</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>工号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>电话：</string>
      </property>
     </widget>
     <widget class="QLabel" name="adm_ano">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>label8</string>
      </property>
     </widget>
     <widget class="QLabel" name="adm_aname">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>label8</string>
      </property>
     </widget>
     <widget class="QLabel" name="adm_sex">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>label8</string>
      </property>
     </widget>
     <widget class="QLabel" name="adm_tel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>250</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>label8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update_password">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>140</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Devanagari</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>修改密码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="adm_password">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Administer,Welcome!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>图书馆信息管理系统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>quitButton</zorder>
   <zorder>Adm_tab</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
